src/CMakeLists.txt

```
@@ -0,0 +1,79 @@
+add_subdirectory(vmec_interface)
+
+
+add_executable(stella.x  ${SOURCES} )
+
+#set flags here
+#TODO add the most common flags needed for stella
+
+set(dialect "-ffree-line-length-none -fno-backslash -I /usr/include")
+set(bounds "-fbounds-check")
+set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} ${dialect} -march=native")
+set(CMAKE_Fortran_FLAGS_DEBUG "${CMAKE_Fortran_FLAGS_DEBUG} ${bounds} -pedantic -g -Wall -fimplicit-none")
+if(PRECISION STREQUAL "DOUBLE")
+	set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fdefault-real-8 -fdefault-double-8")
+endif()
+
+#get package includes and libs here
+
+find_package(MPI REQUIRED)
+if(MPI_FOUND)
```

We'll get an error if MPI isn't found, due to the `REQUIRED`, so we don't need to check `if(MPI_FOUND)`:

```suggestion
find_package(MPI REQUIRED)
```

This applies to the rest of the dependencies too, unless they're not `REQUIRED`